test/cases/testdata/time/test-time-0975.yaml

```diff
@@ -0,0 +1,14 @@
+cases:
+- data:
+  modules:
+  - |
+    package generated
+
+    p = __local0__ {
+      time.diff([time.now_ns()+60*1000*1000*1000, "UTC"], [time.now_ns(), "Local"], __local1__)
```

Comment:
I wonder if this test will fail for others not having "CEST" as Local.

Reply:
I will add something else than "Local", e.g.:
```
./opa_darwin_amd64 eval 'different_tz := time.diff([time.now_ns()+60*1000*1000*1000, "UTC"], [time.now_ns(), "Asia/Shanghai"])' --format=pretty
+---------------+
| different_tz  |
+---------------+
| [0,0,0,0,1,0] |
+---------------+
```